src/Auth0.OidcClient.WinForms/WebViewBrowser.cs

```diff
@@ -47,11 +47,11 @@ public Task<BrowserResult> InvokeAsync(BrowserOptions options, CancellationToken
             var tcs = new TaskCompletionSource<BrowserResult>();
 
             var window = _formFactory();
-            var webView = new WebViewCompatible { Dock = DockStyle.Fill };
+            var webView = new WebView2 { Dock = DockStyle.Fill };
 
-            webView.NavigationCompleted += (sender, e) =>
+            webView.NavigationStarting += (sender, e) =>
```

Comment:
Change back to NavigationStarting because NavigationCompleted does not have Uri property in WinForms.